From a1e233f076b1578f028b2d23b672d9c1195683c3 Mon Sep 17 00:00:00 2001
From: Kai-Uwe Hermann <kai-uwe.hermann@pionix.de>
Date: Tue, 9 Jul 2024 14:46:00 +0200
Subject: [PATCH] Set PYTHON_MODULE_EXTENSION to .so to avoid polluting it with
 host arch

Signed-off-by: Kai-Uwe Hermann <kai-uwe.hermann@pionix.de>
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9c0b86c..da8678d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -121,6 +121,7 @@ endif()
 
 # everest python wrapper
 if (EVEREST_ENABLE_PY_SUPPORT)
+    set(PYTHON_MODULE_EXTENSION ".so")
     add_subdirectory(everestpy)
 endif()
 
